wallet/wallet.go

```diff
@@ -4730,3 +4730,55 @@ func Open(ctx context.Context, cfg *Config) (*Wallet, error) {
 
 	return w, nil
 }
+
+// getCoinjoinTxsSumbByAcct returns a map with key representing the account and
+// the sum of coinjoin output transactions from the account.
+func (w *Wallet) getCoinjoinTxsSumbByAcct(ctx context.Context) (map[uint32]int, error) {
+	const op errors.Op = "wallet.getCoinjoinTxsSumbByAcct"
+	coinJoinTxsByAcctSum := make(map[uint32]int)
+	err := walletdb.View(ctx, w.db, func(dbtx walletdb.ReadTx) error {
+		addrmgrNs := dbtx.ReadBucket(waddrmgrNamespaceKey)
+		txmgrNs := dbtx.ReadBucket(wtxmgrNamespaceKey)
+		// Get current block.  The block height used for calculating
+		// the number of tx confirmations.
+		_, tipHeight := w.txStore.MainChainTip(txmgrNs)
+		rangeFn := func(details []udb.TxDetails) (bool, error) {
+			for _, detail := range details {
+				isMixedTx, _, _ := PossibleCoinJoin(&detail.MsgTx)
+				if !isMixedTx {
+					continue
+				}
+				for _, output := range detail.MsgTx.TxOut {
+					_, addrs, _, _ := txscript.ExtractPkScriptAddrs(output.Version,
+						output.PkScript, w.chainParams)
+					if len(addrs) == 1 {
+						acct, err := w.manager.AddrAccount(addrmgrNs, addrs[0])
+						// mixed output belongs to wallet.
+						if err == nil {
+							coinJoinTxsByAcctSum[acct]++
+						}
+					}
```

Comment:
This will incorrectly confuse the mixed and change account since the change output can appear before the outputs belonging to the wallet's mixed account.  Don't discard the return value for the mix denomination, and compare every output to this value at the top of the loop over outputs, continuing on any other amount.

Reply:
added a denom verification